EIPS/eip-6774.md

```diff
@@ -0,0 +1,262 @@
+---
+eip: 6774
+title: Non-Sellable Tokens
+description: Prevent speculation on NFTs by replacing transfers by a barter mechanism
+author: Cédric Nicolas (@cedric-n-icolas), Casey <casey@logion.community>, Kerel Verwaerde (@NSTKerel), Matthieu Chassagne <Matthieu@amcconsults.com>, Perrin (@pgrandne), Rafael Fitoussi (@fi2c), Raphael (@RaphaelHardFork), Virgil (@virgilea2410)
+discussions-to: https://ethereum-magicians.org/t/eip-6774-non-sellable-tokens/13528
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-03-15
+requires: 712, 721
+---
+
+## Abstract
+
+This EIP is an extension of [ERC-721](./eip-721.md) to create non-sellable tokens (NST). It proposes a strong restriction on transfers in order to prevent speculation on token price while maintaining the possibility of transfers, transfers are performed in a barter way (send to receive or receive to send) in an ecosystem controlled by the creator.
```

Comment:
I'd appreciate a tiny bit of technical detail about _how_ this proposal prevents selling.